terragrunt/aws/central_account/s3.tf

```diff
@@ -91,6 +91,30 @@ data "aws_iam_policy_document" "log_archive_bucket" {
   }
 }
 
+#
+# Bucket Policy that allows the specified principal to get objects
+#
+
+resource "aws_s3_bucket_policy" "log-archive-bucket-get-objects" {
+  bucket = module.log_archive_bucket.s3_bucket_id
+  policy = data.aws_iam_policy_document.log-archive-bucket-get-objects.json
+}
+
+data "aws_iam_policy_document" "log-archive-bucket-get-objects" {
+  statement {
+    principals {
+      type        = "AWS"
+      identifiers = [sensitive(var.cbs_principal_role_arn)]
+    }
+    effect  = "Allow"
+    actions = ["s3:GetObject"]
```

Comment:
Do you just want them to get Object? You might want to add GetObjects and ListObjects so that they can see the bucket contents. 
I have not kept track of this tbh, so I might be mistaken and you might just want to tie it to GetObject but just wanted to throw it in there.

Reply:
Right now, the way their diagram works is that they use the event to know what object to pull so they technically wouldn't need to have any other permissions, as far as I understand :)

Reply:
Thanks for the explanation!